libs/output-mapping/src/Storage/TableStructureValidator.php

```diff
@@ -47,15 +47,15 @@ public function validateTable(string $tableId, array $schemaColumns): TableChang
         }
 
         if ($table['isTyped']) {
-            $missingColumns = $this->validateColumnsName(
+            $tableChangesStore = $this->validateColumnsName(
                 $table['id'],
                 array_map(
                     fn($column) => $column['name'],
                     $table['definition']['columns'],
                 ),
                 $schemaColumns,
+                $tableChangesStore,
             );
-            array_walk($missingColumns, fn($column) => $tableChangesStore->addMissingColumn($column));
```

Comment:
? to by tu melo zustat ne?

Reply:
přesunul jsem to dovnitř tý metody
https://github.com/keboola/platform-libraries/pull/290/files#diff-1279d9968277d98e6ffcce90317881acae7b6f07ca930969877edd2bbbec0b77R235